apps/meteor/app/oembed/server/server.ts

```diff
@@ -287,16 +288,25 @@ const rocketUrlParser = async function (message: IMessage): Promise<IMessage> {
 	log.debug('Parsing message URLs');
 	if (Array.isArray(message.urls)) {
 		log.debug('URLs found', message.urls.length);
+
+		if (
+			message.attachments ||
```

Comment:
is this a breaking change? I mean, if I understood correctly we're not generating URL previews if there are "attachments" on the message, is that correct?

Reply:
Not sure if we should consider this a breaking change since this behavior is currently inconsistent. For quote attachments (for example), we only display previews of links placed before the attachment link. Also, when sending messages with file attachments in the UI, previews are not displayed even after editing the message.